### PR TITLE
Feat/dev updates

### PR DESCRIPTION
**Pull request by IljaDaderko**
- Removed unused / un-necessary dependencies
- Updated rest of the deps
- Simplified / aligned next configs, eslint configs etc.
- Updated everything to work with next 13
- Removed few code comments / unused code
- Fixed hydration error during dev
- Made landing page responsive
- Introduced `useSnapshot` from valtio to react to state changes on ThemeCtrl

### CODE REVIEW

File: src/controllers/ThemeCtrl.tsx [hunk 1]
```diff
@@ -2,13 +2,21 @@ import { proxy, subscribe as valtioSub } from 'valtio/vanilla'
...
+    | 'magenta'
+    | 'orange'
+    | 'purple'
+    | 'teal'
 }
 
 export const initialThemeState = {
-  theme: 'light',
-  accentColor: 'default'
+  theme: 'light' as const,
```
Comment (crypblizz8) [inline src/controllers/ThemeCtrl.tsx:18]: On TS: is there a way for us to indicate it is `as const ` from package level?

I.e. seeing this on modalConfig where I need to always specify `as const` for `modalConfig`?

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -1,71 +1,43 @@
...
 if (!process.env.NEXT_PUBLIC_PROJECT_ID)
   throw new Error('You need to provide NEXT_PUBLIC_PROJECT_ID env variable')
 
 function App({ Component, pageProps }: AppProps) {
-  // ToDo: State for AlphaBanner
-
-  const [theme, setTheme] = useState('')
-  const [accentColor, setAccentColor] = useState('')
-  const [closeBanner, setCloseBanner] = useState(false)
+  const { accentColor, theme } = useSnapshot(ThemeCtrl.state)
```
Comment (crypblizz8) [inline src/pages/_app.tsx:14]: 🤝

**Comment by IljaDaderko** [comment]
@Tom yeh go for it 👍 I think it might be possible to export those `as const` like `const enums` from ts (will have a look at it at some point for modal).

**Comment by crypblizz8** [comment]
Latest Fixes on Image: https://e9f502ec.web3modal-docs.pages.dev/

Will add a ES Lint / more protective measures for Github Actions. @IljaDaderko

**Pull request merged**